setup/src/Magento/Setup/Console/Command/DeployStaticContentCommand.php

```diff
@@ -58,31 +60,47 @@ class DeployStaticContentCommand extends Command
      * @var \Magento\Framework\App\State
      */
     private $appState;
+    /**
+     * @var DirectoryList
+     */
+    private $directoryList;
+    /**
+     * @var File
+     */
+    private $driverFile;
 
     /**
      * StaticContentCommand constructor
      *
-     * @param InputValidator        $inputValidator
-     * @param ConsoleLoggerFactory  $consoleLoggerFactory
-     * @param Options               $options
+     * @param InputValidator $inputValidator
+     * @param ConsoleLoggerFactory $consoleLoggerFactory
+     * @param Options $options
      * @param ObjectManagerProvider $objectManagerProvider
+     * @param DirectoryList $directoryList
+     * @param File $driverFile
+     * @throws \Magento\Setup\Exception
      */
     public function __construct(
         InputValidator $inputValidator,
         ConsoleLoggerFactory $consoleLoggerFactory,
         Options $options,
-        ObjectManagerProvider $objectManagerProvider
+        ObjectManagerProvider $objectManagerProvider,
+        DirectoryList $directoryList,
```

Comment:
Please, inject new dependencies according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)